Scarab/Util/DisplayErrors.cs

```diff
@@ -95,6 +96,21 @@ public static async Task<bool> DisplayHasNotInstalledDependenciesWarning(string
         return result.HasFlag(ButtonResult.Yes) && !result.HasFlag(ButtonResult.None);
     }
 
+    public static Task<bool> DisplayUninstallDependenciesConfirmation(List<ModSelect> options, bool hasExternalMods)
+    {
```

Comment:
Instead of creating a viewmodel, then the window and a task source do it like this https://github.com/TheMulhima/Scarab/blob/29d288d1043b3f62cbe49c9c17b0399985c2ab0f/Scarab/Util/DisplayErrors.cs#L46-L49

Show dialogue as an overload where you can get the result returned